androidtv/androidtv.py

```diff
@@ -210,8 +210,8 @@ def get_properties(self, get_running_apps=True, lazy=False):
             The current app property, or ``None`` if it was not determined
         media_session_state : int, None
             The state from the output of ``dumpsys media_session``, or ``None`` if it was not determined
-        device : str, None
-            The current playback device, or ``None`` if it was not determined
+        audio_output_device : str, None
+            The current audio playback device, or ``None`` if it was not determined
```

Comment:
Could you please change all the occurrences of "current playback device" to "current audio playback device" as you did here.